nengo_loihi/builder/tests/test_nengo_dl.py

```diff
@@ -42,3 +43,13 @@ class MockNoiseBuilder:
     # Second time warns
     with pytest.warns(UserWarning, match="already has a builder"):
         builder_nengo_dl.NoiseBuilder.register(int)(MockNoiseBuilder)
+
+
+def test_install_on_instantiation():
+    nengo_dl = pytest.importorskip("nengo_dl")
+
+    assert LoihiLIF not in nengo_dl.neuron_builders.SimNeuronsBuilder.TF_NEURON_IMPL
```

Comment:
Could there be any weird test order stuff with this, e.g. if another test has already run that's installed the builder?